### ISSUE TITLE
Bug: error occurred when <current_dir> is specified in prefix path patterns

### ISSUE THREAD

**Issue by hdykokd** [tadashi-aikawa/obsidian-another-quick-switcher#140]
Version: 8.4.0

Settings: 
![CleanShot 2023-01-20 at 18 14 20](https://user-images.githubusercontent.com/19975408/213659312-94e65b26-401d-4179-8c7e-182157d21c2f.png)

errror logs in console

```js
Command failed to execute:  obsidian-another-quick-switcher:search-command_search-file-in-current-directory
app.js:1 TypeError: Cannot read properties of undefined (reading 'getActiveFile')
    at getCurrentDirPath (plugin:obsidian-another-quick-switcher:1550:29)
    at String.replace (<anonymous>)
    at eval (plugin:obsidian-another-quick-switcher:2401:18)
    at Array.map (<anonymous>)
    at AnotherQuickSwitcherModal.prefilterItems (plugin:obsidian-another-quick-switcher:2400:41)
    at AnotherQuickSwitcherModal.indexingItems (plugin:obsidian-another-quick-switcher:2344:30)
    at new AnotherQuickSwitcherModal (plugin:obsidian-another-quick-switcher:2256:10)
    at showSearchDialog (plugin:obsidian-another-quick-switcher:3608:17)
    at Object.callback (plugin:obsidian-another-quick-switcher:3698:11)
    at XN (app.js:1:1502974)
```

**Comment by tadashi-aikawa** [comment]
Hi, @hakonhagland :)

Thank you for your bug report. I can reproduce it 👍

**Comment by tadashi-aikawa** [comment]
@hdykokd 
Fixed in [v8.4.1](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/8.4.1) 🚀

**Comment by hdykokd** [comment]
@tadashi-aikawa 
I've updated the plugin and confirmed that works fine. Thank you for dealing with it so quickly!
